routes/user.rb

```diff
@@ -0,0 +1,26 @@
+
+class App < Sinatra::Base
+  get '/users' do
+    User.order(:id).limit(20).to_json
+  end
+  
+  get '/userrank' do
+    User.order(point: :desc).limit(10).to_json
+  end
+
+  get '/getuser/:user_id' do
+    User.find(params[:user_id]).to_json
+  end
+
+  get '/getjoinedspeeches/:user_id' do
+    User.find(params[:user_id]).joined_speeches.to_json
+  end
+
+  get '/getspeeches/:user_id' do
+    User.find(params[:user_id]).speeches.to_json
+  end
+
+  get '/getattendedspeeches/:user_id' do
+    User.find(params[:user_id]).attended_speeches.to_json
+  end
+end
```

Comment:
Moreover, you don't need to merge routes in this PR.

If you need more time to correct the URLs, please move out it from the commits and we can only merge models in this PR.